tests/controllers/excludename/excludename/chainsaw-test.yaml

```diff
@@ -0,0 +1,115 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Authors of Nimbus
+
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: exclude-names-add-csib
```

Comment:
Please use meaningful name and follow convention to prefix test name with its controller name.
For ref: https://github.com/5GSEC/nimbus/blob/5a4217460ea4adb01a0a1afab9e4634b7b924e1a/tests/controllers/securityintentbinding/create/chainsaw-test.yaml#L7